README.md

```diff
@@ -42,6 +42,8 @@ The following [pluggable extras](dist/extras) are provided which can be dropped
 * [Named exports](dist/extras/named-exports.js) convenience extension support for global and AMD module formats (`import { x } from './global.js'` instead of `import G from './global.js'; G.x`)
 * [Named register](dist/extras/named-register.js) supports `System.register('name', ...)` named bundles which can then be imported as `System.import('name')` (as well as AMD named define support)
 * [Transform loader](dist/extras/transform.js) support, using fetch and eval, supporting a hookable `loader.transform`
+* [Import use default](dist/extras/import-use-default.js) provides a convenience interop for AMD modules to return the direct AMD
```

Comment:
Perhaps just `Use default`? (extras are in alphabetical order)

Reply:
Should I rename the file, too, to `use-default.js`? I'll do that.